src/components/App.js

```diff
@@ -1,11 +1,20 @@
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
 
-const App = () => (
-  <>
-    {<Display />}
-    {<ButtonPanel />}
-  </>
-);
+const App = () => {
+  const value = calculate({
+    total: 1,
+    next: 1,
+    operation: '+',
+  }, '=');
```

Comment:
I don't think you need this... if you're following the step-by-step instructions in pathwright, you're only required to import the `calculate.js` function for now.

If I may ask, I'm curious why do you add it?

Reply:
Hi, thanks for your review. 
If I import 'calculate.js' without using it, I get a Linter error.
So I added that to test the method and to use the import to avoid the linter error.

Reply:
Remember to remove it in the next milestone.